src/@orb.yml

```diff
@@ -22,33 +22,28 @@ examples:
           jobs:
             - ggshield/scan:
                 name: ggshield-scan # best practice is to name each orb job
-                base_revision: << pipeline.git.base_revision >>
+                base_revision: <<pipeline.git.base_revision>>
                 revision: <<pipeline.git.revision>>
 
-executors:
-  ggshield-executor:
-    docker:
-      - image: gitguardian/ggshield:<<parameters.version>>
-    parameters:
-      version:
-        type: string
-        default: "latest"
-commands:
-  scan-command:
-    steps:
-      - run: ggshield scan ci
+
 jobs:
   scan:
     parameters:
       base_revision:
         description: pipeline base_revision
         type: string
+        default: ""
       revision:
         description: pipeline revision
         type: string
+      tag:
+        description: Pick a specific gitguardian/ggshield image variant: https://hub.docker.com/r/gitguardian/ggshield/tags
+        default: latest
+        type: string
+    docker:
+      - image: gitguardian/ggshield::<<parameters.tag>>
     environment:
       CIRCLE_RANGE: << parameters.base_revision >>...<<parameters.revision>>
-    executor: ggshield-executor
     steps:
       - checkout
-      - scan-command
+      - ggshield scan -v ci
```

Comment:
When invoking command-line programs you have to use run: https://circleci.com/docs/2.0/configuration-reference/#run.

The way it is right now probably won't work.